network/src/peers.rs

```diff
@@ -0,0 +1,192 @@
+use super::*;
+use async_trait::async_trait;
+use eyre::{eyre, Result};
+use serde_tc::http::*;
+use serde_tc::{serde_tc_full, StubCall};
+use simperby_core::serde_spb;
+use simperby_core::BlockHeader;
+use simperby_core::FinalizationInfo;
+use std::collections::BTreeMap;
+use std::sync::Arc;
+use tokio::fs::File;
+use tokio::io::AsyncReadExt;
+use tokio::io::AsyncWriteExt;
+use tokio::sync::RwLock;
+
+#[derive(Debug)]
+struct PeerStorage {
+    path: String,
+}
+
+impl PeerStorage {
+    pub async fn new(path: &str) -> Result<Self> {
+        Ok(Self {
+            path: path.to_owned(),
+        })
+    }
+
+    pub async fn write(&mut self, peers: Vec<Peer>) -> Result<()> {
+        let _ = tokio::fs::remove_file(&self.path).await;
```

Comment:
Why `let _` binding is used instead of `?` operator?

Reply:
This means that we won't care about the result of the operation (because it might try to remove a file that doesn't exist)